core/src/datayoga_core/job.py

```diff
@@ -125,6 +125,49 @@ def transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> Job
 
         return result
 
+    async def async_transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> JobResult:
+        """
+        Transforms data(async version)
```

Comment:
Add space and period:
Transforms data (async version).

Reply:
✅